apps/api/test/seeders/stripe-transaction.seeder.ts

```diff
@@ -0,0 +1,31 @@
+import { faker } from "@faker-js/faker";
+
+import type { Transaction } from "@src/billing/http-schemas/stripe.schema";
+
+import { generatePaymentMethod } from "@test/seeders/payment-method.seeder";
+
+export class StripeTransactionSeeder {
+  static create({
```

Comment:
suggestion(non-blocking): please read this https://github.com/akash-network/console/discussions/914